src/Roslyn.Suppressions/Roslyn.Suppressions/SupportedSuppressionBuilder.g.cs

```diff
@@ -48,7 +48,7 @@ private static ImmutableArray<SuppressionDescriptor> GetDescriptors()
                 // If the number of diagnostic ids changes significantly or decreases when the analysers are updated,
                 // investigate! There may be a problem with code generator.
                 // *************************************************************************************************
-                // Number of unique diagnostic ids (C# and VB.NET): 494
+                // Number of unique diagnostic ids (C# and VB.NET): 487
```

Comment:
why did this change?

Reply:
I don't know, it's auto generated. I'll look into it to see if there's a problem.

Reply:
I wasn't able to auto generate again so manually changed the file with the correct version.